build/package/generate-msi.sh

```diff
@@ -45,9 +47,7 @@ env GOOS=windows GOARCH=amd64 go build \
 
 go-msi make --path "${WIX_MANIFEST_FILE}" --msi "dist/${PACKAGE_NAME}" --version "${VERSION_GIT}"
 
-go run ./tools/sign -file "dist/${PACKAGE_NAME}"
-
 if [[ "${TOOL_NAME:?}" == atlascli ]]; then
-	go run ./tools/chocolateypkg/chocolateypkg.go -version "${VERSION_GIT}"
+	go run ./tools/chocolateypkg/chocolateypkg.go -version "${VERSION_NAME}"
```

Comment:
at this step, is the msi signed?

Reply:
no, but it does not matter given we do not send msi in choco pack, only an URL.